src/exports.c

```diff
@@ -123,6 +207,72 @@ static void _obj_free(void **obj)
     *obj = NULL;
 }
 
+static bool
+_action_store(UNUSED bool filter_ret, UNUSED json_object *found,
+    UNUSED Needles current)
+{
+    return true;
+}
+
+static bool
+_action_store_true(bool filter_ret, UNUSED json_object *found,
+    UNUSED Needles current)
+{
+    return filter_ret;
+}
+
+static bool
+_action_discard_false(bool filter_ret, UNUSED json_object *found,
+    UNUSED Needles current)
+{
+    return filter_ret;
+}
```

Comment:
Don't see any difference between those functions.

Reply:
That was for symmetry. Good find.

Reply:
i think you united the wrong ones. `_action_discard_false` and `_action_discard_true` actually were opposite, but `_action_store_true` and `_action_discard_false` are the same and one of them may be removed.

Reply:
Blimey. Embarrassing. I think it's better to have redundancy than to squash these.